media.py

```diff
@@ -722,20 +731,20 @@ def getPixel(pic, x, y):
     if (x < 0 or x > pic.getWidth() or y < 0 or y > pic.getHeight()):
         print("The pixel location you chose was out of bounds")
         raise ValueError
-    loc = (x,y)
-    im = pic.getImage()
-    return im.getPixel(loc)
+    return pic.getPixel(x,y)
 
 # Added as a better name for getPixel
 def getPixelAt(pic, x, y):
     return getPixel(pic, x, y)
 
 
 def setRed(pixel, value):
-    value = Pixel.correctLevel(value)
     if not isinstance(pixel, Pixel):
```

Comment:
In all set color functions, use Pixel.correctLevel() to map pixel values to 0..255